### PR TITLE
Integrate tcs

### PR DESCRIPTION
**Pull request by alessiovignoli**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/multiplesequencealign/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/multiplesequencealign _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nf-test test main.nf.test -profile test,docker`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by alessiovignoli** [comment]
Description:
The module TCS for tcoffee has been integreted in the pipeline. So that this evaluation metrics can also be run if specified. It is consistent with other eveluation metrics are already handled. Added to multiQC report.
WARNING the optional input of lib file has bee hardcode to empty. so no lib file can be passed to the TCS module. This can be added in later Isssue.

**Comment by alessiovignoli** [comment]
@nf-core-bot fix linting

File: subworkflows/local/evaluate.nf [hunk 1]
```diff
@@ -18,10 +20,11 @@ workflow EVALUATE {
 
     main:
 
-    ch_versions = Channel.empty()
-    sp_csv = Channel.empty()
-    tc_csv = Channel.empty()
-    irmsd_csv = Channel.empty()
+    ch_version   = Channel.empty()
```
Comment (luisas) [inline subworkflows/local/evaluate.nf:23]:
```suggestion
    ch_versions   = Channel.empty()
```

File: subworkflows/local/evaluate.nf [hunk 2]
```diff
@@ -100,16 +103,40 @@ workflow EVALUATE {
...
 
 
+    // -------------------------------------------
+    // intrinsic evaluation metrics
+    // -------------------------------------------
+
+    // TCS
+    if( params.calc_tcs == true){
+        // the second argument is empty but a lib file can be fed to it
+        TCOFFEE_TCS(ch_msa, [[], []])
```
Comment (luisas) [inline subworkflows/local/evaluate.nf:113]:
```suggestion
        TCOFFEE_TCS(ch_msa, [[:], []])
```

**Review by luisas — APPROVED** [review]
Just 2 minor suggestions

**Pull request merged**